tests/perf/long_runner.py

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import sys
+import time
+from pathlib import Path
+from threading import Thread
+from typing import List
+
+from snowflake.snowpark import Session, Window
+from snowflake.snowpark.functions import avg, col, random as snowflake_random
+
+connection_parameters_path = str(Path(__file__).absolute().parent.parent)
+sys.path.append(connection_parameters_path)
+from parameters import CONNECTION_PARAMETERS  # noqa: E402
+
+logger = logging.getLogger("long-running")
+logger.setLevel(logging.INFO)
+
+
+def generate_columns(n: int) -> List[str]:
+    return [f'{"a" * 50}{i}' for i in range(n)]
+
+
+def run(session: Session, number_of_columns: int):
+    """
+    These APIs are called:
+    simplifier or no simplifier
+    select, sort, and filter
+    with_column
+    aggregate
+    window
+    join
+    union
+    collect
+    to_pandas
+    cache_result
+    to_local_iterator
+    to_pandas_batch
```

Comment:
should we add 1) async api 2) save_as_table 3) write_pandas, which seem important and some users complained before?